src/main/java/build/buildfarm/operations/finder/EnrichedOperationBuilder.java

```diff
@@ -98,13 +114,15 @@ private static Operation jsonToOperation(String json) {
             .ignoringUnknownFields();
 
     if (json == null) {
+      logger.log(Level.SEVERE, "Operation Json is empty");
```

Comment:
Is this a critical error that will error out an operation and/or the build? Is this something that may be better logged as a warning?

Reply:
For context,  this code is used when an individual is looking up operations with `//src/main/java/build/buildfarm:bf-find-operations`.  It won't run normally in production builds.  Switched this to warning.  Ideally anything that will also print to the user's console.